src/main/webapp/includes/homepage-header.jsp

```diff
@@ -9,8 +9,8 @@
       <a class="link nav-item mx-3">Post a recipe</a>
       <c:choose>
              <c:when test="${isLoggedIn == 1}">
-                  <a class="link nav-item mx-3">Welcome ${username}</a>
-                  <a href="${logoutURL}" class="link nav-item mx-3">Log out</a>
+                  <a href="/user?id=${id}" class="link nav-item mx-3">my profile</a>
```

Comment:
Try to keep the word capitalization consistent here and other link changes :-)

Reply:
Will take care of this in a separate Code Cleaning PR